src/sync/kafka/kafka-consumer.ts

```diff
@@ -28,11 +29,12 @@ export class KafkaConsumer implements OnModuleInit {
     onModuleInit() {
         this.topic = process.env.KAFKA_TOPIC || 'sync';
         const config = this.kafkaConfiguration.config;
+
         this.logger.log(`Connecting Kafka consumer to host ${config.kafkaHost}.`);
         if (config['ssl']) {
             this.logger.log(`Consumer TLS is enabled.`);
         } else {
-            this.logger.log(`Consumer TLS is disabled. Supply a certificate and password.`);
+            this.logger.log(`Consumer TLS is disabled. Supply a certificate and password to use TLS.`);
```

Comment:
INFO level of WARN level?

Reply:
Info is prima denk ik. Ik gebruik het om lokaal te testen nog (met een lokale mini cluster)